src/main/java/org/folio/service/manager/input/InputDataManagerImpl.java

```diff
@@ -165,12 +168,49 @@ private void finalizeExport(ExportPayload exportPayload, ExportResult exportResu
     FileDefinition fileExportDefinition = exportPayload.getFileExportDefinition();
     String jobExecutionId = fileExportDefinition.getJobExecutionId();
     String tenantId = exportPayload.getOkapiConnectionParams().getTenantId();
-    jobExecutionService.updateJobStatusById(jobExecutionId, getJobExecutionStatus(exportResult), tenantId);
-    updateFileDefinitionStatusByResult(fileExportDefinition, exportResult, tenantId);
+    JobExecution.Status status = getJobExecutionStatus(exportResult);
```

Comment:
Also the getJobExecutionStatus is likely not necessary right? All that is doing is checking the status based on the last export that happened. Which is not required because we are deciding it based on the errors.

Reply:
we need this status to check if it is completed. then new logic will be called. If status not completed, we don`t need it, cause error logs are present and user will able to open it, and we don`t need to do a call to the DB

Reply:
ok yea makes sense, in case the last batch has failed or completed with error we save a DB call